auth-api/src/auth_api/models/__init__.py

```diff
@@ -21,7 +21,7 @@
 from sbc_common_components.tracing.db_tracing import DBTracing  # noqa: I001, I004
 
 from .account_login_options import AccountLoginOptions
-from .account_payment_settings import AccountPaymentSettings
+from .account_payment_settings import AccountPaymentSettings as AccountPaymentSettingsDeprecated
```

Comment:
Why call 'deprecated'?

Reply:
hm..the model is no longer used... we removed the AccountPaymentSettings from auth side .. I need the model for migration ... 
just to make sure nobody else is planning to use that model...not really necessary or there could be better way to do this...i am open to ideas